src/equations/compressible_navier_stokes_3d.jl

```diff
@@ -0,0 +1,380 @@
+@doc raw"""
```

Comment:
Did you check the rendering of the new docstrings?

Reply:
They look good, but there is a typo that I need to fix.

Reply:
Yes, that typo was fixed.